unifi.js

```diff
@@ -1929,6 +1929,17 @@ class Controller extends EventEmitter {
     return this._request('/api/s/<SITE>/list/portforward');
   }
 
+  /**
+   * Set port forwarding rule - set_port_forwarding()
+   *
+   * required parameter <payload> = stdClass object or associative array containing the configuration to create/update the forwarding
+   *                                rule. Must be a (partial) object structured in the same manner as is returned by getPortForwarding().
+   * return true on success
+   */
+  setPortForwarding(rule_id, payload) {
+    return this._request('/api/s/<SITE>/rest/portforward/' + rule_id.trim(), payload, 'PUT');
```

Comment:
I am not sure if we should generally allow a whole payload to be submitted via `/rest/portforward` but limit it to enabled/disabled state like this is mentioned in your referenced documentation (https://ubntwiki.com/products/software/unifi-controller/api#update_of_port_forward_rules). There the authors say that 

```
{
    "enabled": true
}
```

seem to be fine to be sent. But allowing to send arbitrary payload seems to open way too much generalism, IMHO. Or do you have any more experience on which payload can be sent via `PUT`?